modules/extension/ysld/src/main/java/org/geotools/ysld/parse/RuleParser.java

```diff
@@ -49,14 +50,17 @@ public void handle(YamlObject<?> obj, YamlParseContext context) {
             featureStyle.rules().add(rule);
 
             rule.setName(r.str("name"));
-            if (r.has("title")) {
-                rule.setTitle(r.str("title"));
-            }
-            if (r.has("abstract")) {
-                rule.setAbstract(r.str("abstract"));
-            }
```

Comment:
This change would be better off in a separate commit as it is really just housekeeping unrelated to the other change.

Reply:
These unnecessary code changes have been removed.